src/YoutubeIframe.js

```diff
@@ -219,15 +241,10 @@ const YoutubeIframe = (props, ref) => {
     const data = ytScript.urlEncodedJSON;
 
     return {uri: base + '?data=' + data};
-  }, [
-    videoId,
-    playList,
-    useLocalHTML,
-    contentScale,
-    baseUrlOverride,
-    allowWebViewZoom,
-    initialPlayerParams,
-  ]);
+    // videoId, playlist, initialPlayerParams are only used once when initializing YT.Player instance
+    // further changes are handled by injectJavaScript
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I usually avoid disabling this particular rule, will potentially lead to bugs in the future due to missed dependencies.

since we only want to send the _first_ videoID and playList data, a `ref` can be used to store the values, and this will not change over the course of this components life cycle.

```js
const firstVideoId = useRef(videoId);
const firstPlaylist = useRef(playList);

...

const ytScript = MAIN_SCRIPT(
      firstVideoId.current,
      firstPlaylist.current,
      initialPlayerParams,
      allowWebViewZoom,
      contentScale,
    );
```